source/samples/addressbook.d

```diff
@@ -531,20 +579,21 @@ public:
 	__gshared static SegmentReader b_98808e9832e8bc18 = GeneratedClassSupport.decodeRawBytes([
 	  0x0,0x0,0x0,0x0,0x5,0x0,0x6,0x0,
 	  0x18,0xbc,0xe8,0x32,0x98,0x8e,0x80,0x98,
-	  0x12,0x0,0x0,0x0,0x1,0x0,0x1,0x0,
+	  0x21,0x0,0x0,0x0,0x1,0x0,0x1,0x0,
```

Comment:
I'm a little concerned that this has changed. I don't see how this change could cause this. Perhaps I forgot to update the pregenerated files after an earlier update.